src/ekomark/benchmark/external/apfel_utils.py

```diff
@@ -34,8 +32,7 @@ def compute_apfel_data(
         ref : dict
             output containing: target_xgrid, values
     """
-
-    target_xgrid = operators["xgrid"]
+    target_xgrid = operators["interpolation_xgrid"]
```

Comment:
Is this correct?

Reply:
I would say no: it is the old standard, that has already been updated, and here reverted.

Most likely, it has been tested with something old, e.g. `ekomark` may still be old-styled.

Reply:
I have changed it since ekomark was crashing after the merge. I think we should change ekomark to include the new name

Reply:
yes might be that this is the case

Reply:
> I have changed it since ekomark was crashing after the merge. I think we should change ekomark to include the new name

Since you confirm it was crashing, I'm pretty sure that's the problem.

However, this is nice-to-have, but not really a required/prioritized feature: `ekomark` is not even running in the CI, so if needed can lack behind (but possibly not for too long, it is useful, so we don't want it to be broken).
In case you will merge this PR without recovering `ekomark`, remember to open a dedicated issue, in order to forget about the problem.